components/ledger/internal/adapters/postgres/balance/balance.go

```
@@ -49,10 +54,20 @@ func (b *BalancePostgreSQLModel) FromEntity(balance *mmodel.Balance) {
+		Direction:      balance.Direction,
+		OverdraftUsed:  balance.OverdraftUsed,
 		CreatedAt:      balance.CreatedAt,
 		UpdatedAt:      balance.UpdatedAt,
 	}
 
+	if balance.Settings != nil {
+		// json.Marshal cannot fail for BalanceSettings: it contains only
+		// JSON-safe primitives (string, bool, *string). We intentionally
+		// discard the error and leave b.Settings as nil on the (impossible)
+		// failure path, matching the existing behavior for a nil Settings.
+		b.Settings, _ = json.Marshal(balance.Settings)
```

golangci-lint (2.11.4)

[error] Error return value of `encoding/json.Marshal` is not checked

(errchkjson)